src/app/modules/main/profile_section/notifications/module.nim

```diff
@@ -115,19 +113,14 @@ method onToggleSection*(self: Module, sectionType: SectionType) =
   if(sectionType != SectionType.Community):
     return
 
-  if(singletonInstance.localAccountSensitiveSettings.getCommunitiesEnabled()):
-    let channelGroups = self.controller.getChannelGroups()
-    for cg in channelGroups:
-      if cg.channelGroupType == ChannelGroupType.Community:
-        let ind = self.view.exemptionsModel().findIndexForItemId(cg.id)
-        if(ind != -1):
-          continue
-        let item = self.createItem(cg.id, cg.name, cg.images.thumbnail, cg.color, joinedTimestamp = 0, item.Type.Community)
-        self.view.exemptionsModel().addItem(item)
-  else:
-    for item in self.view.exemptionsModel().modelIterator():
-      if(item.itemType == Type.Community and self.controller.removeNotifSettingExemptions(item.id)):
-          self.view.exemptionsModel().removeItemById(item.id)
+  let channelGroups = self.controller.getChannelGroups()
```

Comment:
@borismelnik if we don't have a feature to enable/disable communities any more, then you're free to remove entire `onToggleSection` proc, and as well connection from the controller's `init` proc.

Reply:
@saledjenic  ok